### PR TITLE
ODH-16889 - updates for Gaudi 3 integration

### PR DESCRIPTION
**Pull request by chtyler**
## Description
These are the updates required for the Gaudi 3 enablement in RHOAI. 

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: modules/enabling-intel-gaudi-ai-accelerators.adoc [hunk 1]
```diff
@@ -4,36 +4,45 @@
...
 
 [role='_abstract']
-Before you can use Intel Gaudi AI accelerators in {productname-short}, you must install the necessary dependencies and deploy the HabanaAI Operator. 
+Before you can use Intel Gaudi AI accelerators in {productname-short}, you must install the required dependencies, deploy the Intel Gaudi AI Accelerator Operator, and configure the environment.
 
 .Prerequisites
 ifdef::upstream,self-managed[]
 * You have logged in to {openshift-platform}.
 * You have the `cluster-admin` role in {openshift-platform}.
+* Intel Gaudi devices are installed and detected in your environment.
```
Comment (eturner24) [inline modules/enabling-intel-gaudi-ai-accelerators.adoc:13]: Is there a way to rewrite to make parallel or would that make it more confusing/wordy? Something like this perhaps?:
"You have installed your Intel Gaudi accelerator and confirmed that it is detected in your environment."

File: modules/intel-gaudi-ai-accelerator-integration.adoc [hunk 2]
```diff
@@ -1,23 +1,37 @@
...
-= Intel Gaudi AI Accelerator integration
+= Intel Gaudi AI Accelerator integration  
 
 [role='_abstract']
-To accelerate your high-performance deep learning (DL) models, you can integrate Intel Gaudi AI accelerators in {productname-short}. This allows your data scientists to use Gaudi libraries and software associated with Intel Gaudi AI accelerators from their workbench. Before you can enable Intel Gaudi AI accelerators in {productname-short}, you must install the necessary dependencies. Also, the version of the Intel Gaudi AI Operator that you install must match the version of the corresponding workbench image in your deployment. However, a workbench image for Intel Gaudi accelerators is not included in {productname-short} by default. Instead, you must create and configure a custom notebook to enable Intel Gaudi AI support. 
+To accelerate your high-performance deep learning (DL) models, you can integrate Intel Gaudi AI accelerators into {productname-short}. This integration enables your data scientists to use Gaudi libraries and software associated with Intel Gaudi AI accelerators through custom-configured workbench instances.
```
Comment (eturner24) [inline modules/intel-gaudi-ai-accelerator-integration.adoc:7]: Do we need the (DL)? We don't abbreviate this again except in the context of EC2 DL1 - and I'm not sure if DL even is the same acronym there.

File: modules/intel-gaudi-ai-accelerator-integration.adoc [hunk 3]
```diff
@@ -1,23 +1,37 @@
...
+Intel Gaudi AI accelerators offer optimized performance for deep learning workloads, with the latest Gaudi 3 devices providing significant improvements in training speed and energy efficiency. These accelerators are suitable for enterprises running machine learning and AI applications on {productname-short}.  
 
-To identify the Intel Gaudi AI accelerators present in your deployment, use the `lspci` utility. For more information, see link:https://linux.die.net/man/8/lspci[lspci(8) - Linux man page].
+Before you can enable Intel Gaudi AI accelerators in {productname-short}, you must complete the following steps:
+
+1. Install version 1.18 of the Intel Gaudi AI Accelerator Operator from OperatorHub.  
+2. Create and configure a custom workbench image for Intel Gaudi AI accelerators. A prebuilt workbench image for Gaudi accelerators is not included in {productname-short}.  
+3. Manually define and configure an accelerator profile for each Intel Gaudi AI device in your environment.
```
Comment (eturner24) [inline modules/intel-gaudi-ai-accelerator-integration.adoc:15]:
```suggestion
. Install version 1.18 of the Intel Gaudi AI Accelerator Operator from OperatorHub.  
. Create and configure a custom workbench image for Intel Gaudi AI accelerators. A prebuilt workbench image for Gaudi accelerators is not included in {productname-short}.  
. Manually define and configure an accelerator profile for each Intel Gaudi AI device in your environment.  
```

File: modules/intel-gaudi-ai-accelerator-integration.adoc [hunk 4]
```diff
@@ -1,23 +1,37 @@
...
+3. Manually define and configure an accelerator profile for each Intel Gaudi AI device in your environment.  
+
+{productname-short} supports Intel Gaudi devices up to Intel Gaudi 3. The Intel Gaudi 3 accelerators, in particular, offer:
+
+* Improved Training Throughput: Reduce the time required to train large models by utilizing advanced tensor processing cores and increased memory bandwidth.  
+* Energy Efficiency: Lower power consumption while maintaining high performance, reducing operational costs for large-scale deployments.  
+* Scalable Architecture: Support for scaling across multiple devices and nodes, making them effective for distributed training configurations on OpenShift.  
+
+Your OpenShift platform must support EC2 DL1 instances to use Intel Gaudi AI accelerators in an Amazon EC2 DL1 instance. Once the accelerators are enabled, the custom workbench image is created, and the accelerator profiles are configured, Intel Gaudi AI accelerators become available for use in workbench instances or model serving.
```
Comment (eturner24) [inline modules/intel-gaudi-ai-accelerator-integration.adoc:23]: Can we make the last sentence active voice?
"You can use Intel Gaudi AI accelerators in workbench instances or model serving after you enable the accelerators, create a custom workbench image, and configure the accelerator profile."

**Review by eturner24 — CHANGES_REQUESTED** [review]

File: modules/enabling-intel-gaudi-ai-accelerators.adoc (hunk 1 above)
Comment (chtyler) [reply modules/enabling-intel-gaudi-ai-accelerators.adoc:13]: Changed to your suggestion.

File: modules/intel-gaudi-ai-accelerator-integration.adoc (hunk 2 above)
Comment (chtyler) [reply modules/intel-gaudi-ai-accelerator-integration.adoc:7]: EC2 DL1 is a separate acronym relating to a different matter. I agree, we don't need the (DL), removing it.

File: modules/intel-gaudi-ai-accelerator-integration.adoc (hunk 3 above)
Comment (chtyler) [reply modules/intel-gaudi-ai-accelerator-integration.adoc:15]: Changed to your suggestion.

File: modules/intel-gaudi-ai-accelerator-integration.adoc (hunk 4 above)
Comment (chtyler) [reply modules/intel-gaudi-ai-accelerator-integration.adoc:23]: Yes. Changed to your suggestion.

**Review by eturner24 — APPROVED** [review]

**Pull request merged**